src/PhpSpreadsheet/Reader/Gnumeric.php

```diff
@@ -266,9 +266,10 @@ public function loadIntoExisting(string $pFilename, Spreadsheet $spreadsheet): S
         (new Properties($this->spreadsheet))->readProperties($xml, $gnmXML);
 
         $worksheetID = 0;
-        foreach ($gnmXML->Sheets->Sheet as $sheet) {
+        foreach ($gnmXML->Sheets->Sheet as $sheetOrNull) {
+            $sheet = self::testSimpleXml($sheetOrNull);
             $worksheetName = (string) $sheet->Name;
-            if ((isset($this->loadSheetsOnly)) && (!in_array($worksheetName, $this->loadSheetsOnly))) {
+            if (is_array($this->loadSheetsOnly) && !in_array($worksheetName, $this->loadSheetsOnly)) {
```

Comment:
The `in_array()` check should also use the 3rd argument for strict checking

Reply:
Thanks, hadn't known about the extra argument.

Reply:
Most of this codebase was written before it was introduced to PHP, but I try to apply it whenever I'm modifying anything and notice an `in_array()`